src/primitive_crypto.cc

```diff
@@ -24,6 +24,45 @@
 
 namespace toit {
 
+enum class AesTypes {
+  Ecb,
+  Cbc
+};
+
+static Object* aes_crypt(Process* process, AesEcbContext* context, Blob* input, int from, int to, bool encrypt, AesTypes type) {
+  if (from < 0 || to > input->length() || from > to || ((to - from) % 16) != 0) INVALID_ARGUMENT;
+
+  Error* error = null;
+  ByteArray* result = process->allocate_byte_array(to - from, &error);
+  if (result == null) return error;
+
+  ByteArray::Bytes output_bytes(result);
+
+  switch (type)
```

Comment:
We typically indent the switches like this:

```
switch (...) {
  case ...:
    ...
    break;
  case ...:
    ...
    break;
}
```

Reply:
The switch has been indented like shown.